pkg/apis/controlplane/v1beta2/sets.go

```diff
@@ -38,13 +38,20 @@ func normalizeGroupMember(member *GroupMember) groupMemberKey {
 	const delimiter = "/"
 	var b strings.Builder
 	if member.Pod != nil {
+		b.WriteString("Pod:")
```

Comment:
we should make pkg/apis/controlplane/sets.go consistent

Reply:
Done.